packages/strapi-plugin-content-manager/admin/src/translations/fr.json

```diff
@@ -21,7 +21,7 @@
   "components.FiltersPickWrapper.hide": "Fermer",
   "components.LimitSelect.itemsPerPage": "Éléments par page",
   "components.Search.placeholder": "Rechercher une entrée...",
-  "components.TableDelete.delete": "Tout supprimer",
+  "components.TableDelete.delete": "Supprimer sélectionnée",
```

Comment:
This should be a little more frenchy
```
Supprimer les éléments sélectionnés
```

Reply:
Like "Supprimer les éléments sélectionnés" for example, we have space we can have a long sentence.

Reply:
/s I do think that's a little long there @soupette